pkg/apis/storage/types.go

```diff
@@ -65,8 +65,19 @@ type StorageClass struct {
 	// for all PVs created from this storageclass.
 	// +optional
 	AllowVolumeExpansion *bool
+
+	// VolumeBindingMode indicates how PersistentVolumeClaims should be
+	// provisioned and bound.  When unset, VolumeBindingImmediate is used.
+	// This field is alpha-level and is only honored by servers that enable
+	// the VolumeTopologyBinding feature.
+	// +optional
+	VolumeBindingMode *VolumeBindingMode
 }
 
+// NoProvisioner is a special Provisioner value to use when a StorageClass
+// is required but there is no dynamic provisioner.
+const NoProvisioner = "k8s.io/no-provisioner"
```

Comment:
Should be `kubernetes.io/no-provisioner` - we don't generally use k8s.io in formal names (except API versions, but that one got away from me).